hera_mc/tests/test_geo_location.py

```diff
@@ -65,13 +65,26 @@ def test_update_new(mcsession, geo_handle):
     geo_location.update(mcsession, data, add_new_geo=True)
     located = geo_handle.get_location([nte], "now")
     assert located[0].station_type_name == "herahex"
+    nte = "HH_another_new_test_element"
+    data = [
+        [nte, "station_name", nte],
+        [nte, "station_type_name", "herahex"],
+        [nte, "datum", "WGS84"],
+        [nte, "tile", "34J"],
+        [nte, "northing", 6600000.0],
+        [nte, "easting", 541000.0],
+        [nte, "elevation", 1050.0],
+        [nte, "created_gpstime", 1172530000],
+    ]
+    pytest.raises(ValueError, geo_location.update, mcsession, data, add_new_geo=False)
```

Comment:
please specify what the error message should be using the `match` parameter. The best would be to do this in a context manger form:
```suggestion
    with pytest.raises(ValueError, match=<expected error message>):
        geo_location.update(mcsession, data, add_new_geo=False)
```